data/ecosystems/a/aptos.toml

```diff
@@ -17,6 +17,7 @@ sub_ecosystems = [
   "BlockPi Network",
   "Bware Labs",
   "Chainbase",
+  "Cellana Finance",
```

Comment:
This is failing a build check due to sort (see [here](https://github.com/electric-capital/crypto-ecosystems/actions/runs/9658311800/job/26639228291?pr=1391). Please move this above Chainbase to maintain alphabetical order.

Reply:
we updated